Lib/test/test_getpass.py

```diff
@@ -187,8 +186,10 @@ def test_uses_msvcrt_directly(self):
             mock_stream.flush.assert_called()
 
     def test_handles_backspace(self):
-        with mock.patch('msvcrt.getch') as getch, \
-             mock.patch('msvcrt.putch') as putch:
+        with (
+        	mock.patch('msvcrt.getch') as getch,
+            mock.patch('msvcrt.putch') as putch,
+		):
```

Comment:
Ah sorry but there are tabs and spaces mixed in this suggestion. I would suggest choosing the same style as the other tests (see test_falls_back_to_stdin). If you choose this style, please also apply it to `test_falls_back_to_fallback_if_msvcrt_raises`